packages/go/project.bri

```diff
@@ -80,13 +99,20 @@ export async function goInstall(
   const modules = goModDownload(options.goModule);
 
   let buildResult = std.runBash`
-    go install
+    # Run generate if requested
+    if [ "${options.buildParams?.generate}" = "true" ]; then
```

Comment:
Nitpick: I'd feel a little better if this were passed as an env var, meaning the code would be `if [ "$go_generate" = "true" ]`. I view it the same way as interpolating a string in a SQL query: it'll usually work and in some cases it will always work, but (in my opinion) it's better practice be consistent and _always_ pass inputs as variables

(eventually I think I might go as far as having a lint warning when interpolating when using `runBash`)

Reply:
I thought for this one that since we control the value (which is just a primitive boolean) I could have passed it directly, but I'll update the bash to add another env var.

Reply:
I updated it in the last commit.